src/Enum.js

```diff
@@ -0,0 +1,29 @@
+export const TIMEZONE_ENUM = {
```

Comment:
there is no enum in javascript. There are objects that have key value pairs. You can simply call it timezones

Reply:
@meenakshi-dhanani , the reason I did this is because the backend respond to this request with the timezone enum `name` 
<img width="910" alt="Screen Shot 2020-07-19 at 10 30 46 pm" src="https://user-images.githubusercontent.com/29667122/87874901-d892ac80-ca10-11ea-9599-a65124573e1c.png">

which is what's being recorded in postgresql.
<img width="524" alt="Screen Shot 2020-07-19 at 10 27 25 pm" src="https://user-images.githubusercontent.com/29667122/87874846-75087f00-ca10-11ea-9899-f10a2dd223f9.png">

<img width="658" alt="Screen Shot 2020-07-19 at 10 32 15 pm" src="https://user-images.githubusercontent.com/29667122/87874857-881b4f00-ca10-11ea-9105-6ce21e6c2efd.png">

However, what I want the user to see is the timezone `value` which according to [this stackoverflow discussion](https://stackoverflow.com/questions/18111657/how-to-get-names-of-enum-entries) could be done in in this way (how I did it in the code).

There is another alternative is to change the backend response and request model to accept/return human-readable format (aka enum value) from the backend side and do the logic to convert it to enum name before saving it to the database. 
But again, seeing how we keep machine format (unix datetime) for datetime on our backend and only change it to human-readable on frontend, I thought there's a convention somewhere (I'm not sure if there is, but @ramitsawhney27 maybe can clarify this for us) to keep backend as close as machine language

<img width="652" alt="Screen Shot 2020-07-19 at 10 31 12 pm" src="https://user-images.githubusercontent.com/29667122/87875025-ad5c8d00-ca11-11ea-9e72-15e76d281cac.png">

Reply:
Choose to keep the mapping logic in the frontend/backend. And follow it consistently. Based on our discussion, keeping it in the backend right now is less effort. So you can continue with that. So frontend will not keep any mappings, as per my understanding

Reply:
Ok. will change it now.